client/stylesheets/_responsive.scss

```diff
@@ -155,12 +238,18 @@
     .homeCardLi {
     	width: 65%;
     }
+    .searchResultsCardLi {
+    	width: 70% !important;
+    }
 }
 
 @media (max-width: 360px) {
     .homeCardLi {
     	width: 70%;
     }
+    .searchResultsCardLi {
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Selector `searchResultsCardLi` should be written in lowercase with hyphens